dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.cdi3.0-appSecurityClient.feature

```diff
@@ -0,0 +1,13 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
```

Comment:
The javaee version of this autofeature had 1.0 after appSecurityClient.  Did you remove it on purpose?

Reply:
Yes. I didn't want to tie it to any version of appSecurityClient, just that app security is enabled.